client/src/components/Footer.js

```diff
@@ -73,22 +94,14 @@ const Footer = () => {
               </Text>
             </Link>
           </Col>
-          <Button
-            style={{
-              width: '15rem',
-              height: '3.5rem',
-              backgroundColor: '#7BCF95',
-              borderColor: '#7BCF95',
-              radius: '3px',
-            }}
-          >
+          <Button href={`https://code4.ro/${currentLanguage}/${donateTxt}`}>
```

Comment:
Do we need to translate terms in the URL ?

Reply:
Yes, the URL`s are different for English and Romanian, but not for Hungarian which will be problematic if we plan to add support for this language as @tudoramariei mentioned.

[https://code4.ro/en/donate](https://code4.ro/en/donate)
[https://code4.ro/ro/doneaza](https://code4.ro/ro/doneaza)
[https://code4.ro/hu/doneaza](https://code4.ro/hu/doneaza)

Reply:
This is really strange to me. Usually, the URLs are not translated, and https://code4.ro/en/donate would be used for all languages.